h5py/h5t.pyx

```diff
@@ -250,6 +249,7 @@ PYTHON_OBJECT = lockid(H5PY_OBJ)
 # Translation tables for HDF5 -> NumPy dtype conversion
 cdef dict _order_map = { H5T_ORDER_NONE: '|', H5T_ORDER_LE: '<', H5T_ORDER_BE: '>'}
 cdef dict _sign_map  = { H5T_SGN_NONE: 'u', H5T_SGN_2: 'i' }
+cdef dict _custom_dtype_map = {}
```

Comment:
Should this be called `_dtype_map`, as there is nothing necessarily custom about them?